lib/resolve.js

```diff
@@ -38,31 +38,31 @@ const buildModMap = (modMap, watchedPaths, rootPackage) => {
   });
 };
 
-const resolveErrorRe = /Cannot find module '(.+)' from '(.+)'/;
```

Comment:
Checked Node's and `node-browser-resolve` source code, did not find `from ...` in messages